packages/bot-skeleton/src/scratch/backward-compatibility.js

```diff
@@ -152,12 +152,12 @@ export default class BlockConversion {
                 statement_blocks: [block],
             };
         };
-        const tradeOptions = (block_node, is_symbol_type = false) => {
+        const trade_options = (block_node, is_symbol_type = false) => {
```

Comment:
@maryia-matskevich-deriv any particular reason why we change tradeOptions function from camel case to snake case?

Reply:
@farabi-deriv Oh yes this is a function, I thought I worked with an object..✅fixed